src/containers/AccessPointDetails/components/OS/components/HighChartGraph/index.js

```diff
@@ -58,16 +57,20 @@ const HighChartGraph = ({ osData }) => {
   }, [osData]);
 
   return (
-    <HighchartsStockChart data-testid="highchartsGraph">
+    <HighchartsStockChart
+      data-testid="highchartsGraph"
+      time={{
+        useUTC: false,
+      }}
+    >
       <Chart zoomType="x" backgroundColor="#141414" />
 
-      <Tooltip split={false} shared useHTML />
+      <Tooltip split={false} shared useHTML xDateFormat="%b %e %Y %l:%M:%S%P" />
```

Comment:
format the tooltip so it shows the freeMemory in MB and cpuUtilized in % and cpuTemperature in °C